Buildings/Obsolete/Controls/OBC/CDL/Logical/TrueHold.mo

```diff
@@ -1,4 +1,4 @@
-within Buildings.Controls.OBC.CDL.Logical;
+within Buildings.Obsolete.Controls.OBC.CDL.Logical;
```

Comment:
add an anotation as `obsolete = "This model is obsolete"`, see the block `Buildings.Obsolete.Controls.OBC.CDL.Logical.TriggeredTrapezoid`

Reply:
Addressed with afb1c60